process_data.sh

```diff
@@ -57,20 +57,24 @@ concatenate_b0_and_dwi(){
 label_if_does_not_exist(){
   local file="$1"
   local file_seg="$2"
+  local file_space_other="$3"
   # Update global variable with segmentation file name
-  FILELABEL="${file}_labels"
-  FILELABELMANUAL="${PATH_DATA}/derivatives/labels/${SUBJECT}/anat/${FILELABEL}-manual.nii.gz"
+  FILELABEL="${file}_label-discs_dlabel" #label-discs_dlabel
+  FILELABELMANUAL="${PATH_DATA}/derivatives/labels/${SUBJECT}/anat/${FILELABEL}.nii.gz"
   echo "Looking for manual label: $FILELABELMANUAL"
   if [[ -e $FILELABELMANUAL ]]; then
     echo "Found! Using manual labels."
     rsync -avzh $FILELABELMANUAL ${FILELABEL}.nii.gz
+    # Generate labeled segmentation from manual disc labels
+    sct_label_vertebrae -i ${file}.nii.gz -s ${file_seg}.nii.gz -discfile ${FILELABEL}.nii.gz -c t1
```

Comment:
This command is actually never exectuted since we have manual disc labels (`$FILELABELMANUAL`) for all subjcets, right?
But, if it was executed, the labeling would be done on the original image before resampling and reorientation (`${file}.nii.gz`). Is it okay? Don't we want to run labeling on `file_space_other` instead?

Reply:
this command is excecuted you mean right? Since we are in the if statement?

Reply:
You are right. Sorry, I misread the condition.